Stripe/StripeiOSTests/STPPaymentIntentFunctionalTest.swift

```diff
@@ -1233,6 +1233,62 @@ class STPPaymentIntentFunctionalTest: XCTestCase {
         waitForExpectations(timeout: STPTestingNetworkRequestTimeout, handler: nil)
     }
 
+    // MARK: - Multibanco
+
+    func testConfirmPaymentIntentWithMultibanco() {
+        var clientSecret: String?
+        let createExpectation = self.expectation(description: "Create PaymentIntent.")
+        STPTestingAPIClient.shared.createPaymentIntent(
+            withParams: [
+                "payment_method_types": ["multibanco"],
+                "currency": "eur",
+                "amount": NSNumber(value: 6000),
+            ],
+            account: "us"
+        ) { createdClientSecret, creationError in
+            XCTAssertNotNil(createdClientSecret)
+            XCTAssertNil(creationError)
+            createExpectation.fulfill()
+            clientSecret = createdClientSecret
+        }
+        waitForExpectations(timeout: STPTestingNetworkRequestTimeout, handler: nil)
+        XCTAssertNotNil(clientSecret)
+
+        let client = STPAPIClient(publishableKey: STPTestingDefaultPublishableKey)
+        let expectation = self.expectation(description: "Payment Intent confirm")
+
+        let paymentIntentParams = STPPaymentIntentParams(clientSecret: clientSecret!)
+
+        let billingDetails = STPPaymentMethodBillingDetails()
+        billingDetails.email = "tester@example.com"
+
+        paymentIntentParams.paymentMethodParams = STPPaymentMethodParams(
+            multibanco: STPPaymentMethodMultibancoParams(),
+            billingDetails: billingDetails,
+            metadata: [
+                "test_key": "test_value",
+            ]
+        )
+
+        let options = STPConfirmPaymentMethodOptions()
+        paymentIntentParams.paymentMethodOptions = options
+        paymentIntentParams.returnURL = "example-app-scheme://unused"
+        client.confirmPaymentIntent(with: paymentIntentParams) { paymentIntent, error in
+            XCTAssertNil(error, "With valid key + secret, should be able to confirm the intent")
+
```

Comment:
might better to:
```
guard let paymentIntent= paymentIntent else {
   XCTFail()
}
```

Reply:
Tried to use `XCTUnwrap` but the complier complains for some reason inside of a closure.